src/site/stages/build/drupal/graphql/CountEntityTypes.graphql.js

```diff
@@ -20,6 +20,17 @@ const CountEntityTypes = `
   	) {
     count
   }
+  
+  vetCenters: nodeQuery(
+    filter: {
+      conditions: [
+        # TODO: uncomment when vet_centers officially go live:
+        # {field: "status", value: ["1"]},
```

Comment:
Should this be uncommented now?

Reply:
I don't think so, because vet centers are still in progress in CMS (unpublished), but we want the queries to paginate anyway.

Reply:
the actual graphql query appears to filter on only published pages, https://github.com/department-of-veterans-affairs/vets-website/pull/16686/files#diff-9cae9bfcc953806c5a54f4685e67b159a971f59418fb635546dc57b1b8d0d6cfR87

Reply:
That query uses `$onlyPublishedContent` variable, just like every other. Am I missing something?

We also have a [PR](https://github.com/department-of-veterans-affairs/vets-website/pull/16713) to use an environment variable to override `$onlyPublishedContent`, BTW.